### PR TITLE
Add popup window

### PR DESCRIPTION
**Pull request by hiromataba**
In this milestone we have added the popup window details to the portfolio

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (descholar-ceo) [inline .vscode/settings.json]: - [ ] We should not commit unrelated files to our repository. I think it should be very nice if you removed this `.vscode` folder from your repository

File: index.js [hunk 2]
```diff
@@ -1,8 +1,147 @@
...
+
+    // Add Skills
+    const skills = document.createElement('ul');
+    skills.classList.add('skills');
+    project.projectTechnologies.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('card-skills-li');
+      techItem.innerText = tech;
+      skills.appendChild(techItem);
+    });
```
Comment (descholar-ceo) [inline index.js]: - [ ] This skills bar is not fitting well in the mobile screen (on width of 320px), it would be very nice if you make them responsive on the screen

File: index.js [hunk 3]
```diff
@@ -1,8 +1,147 @@
...
+  });
+}
+
+implementProjects();
+
+// POP UP WINDOW
+const popupWindow = document.querySelector('.popup-window');
+const seeProjectBtns = [...document.querySelectorAll('.see-project-button')];
+const showPopupWindow = (btn) => {
+  popupWindow.style.display = 'flex';
```
Comment (descholar-ceo) [inline index.js]: - [ ] Kindly keep all of the stylings in a stylesheet file, this will help you in the future when you want to do any update or debug!

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @hiromataba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+    // Add Skills
+    const skills = document.createElement('ul');
+    skills.classList.add('skills');
+    project.projectTechnologies.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('card-skills-li');
+      techItem.innerText = tech;
+      skills.appendChild(techItem);
+    });
```
Comment (Temmarie) [inline js/index.js:91]: - [ ] As requested by the previous reviewer,  The skills bar is not fitting well on the mobile screen (on the width of 320px), it would be very nice if you make them responsive on the screen. I think this has to do with the height of the popup window. 
_See images below for reference._

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @hiromataba     ! :wave: :wave:

#### You have done really well in implementing the previously requested changes.
 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

## Optional improvement ☑️ 

- Don't forget the yellow line 😄
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
